### DISCUSSION TITLE
Is there a way to get the amount of pages of a PDF

### DISCUSSION THREAD

**Discussion by erikradisch** [cantaloupe-project/cantaloupe#461] [Q&A] [answered]
I wonder if can provide me with the information, how many pages a PDF has?

**Comment by adolski** [comment, marked as answer]
Yes, not in 4.1, but in 5.0: https://github.com/cantaloupe-project/cantaloupe/issues/344#issuecomment-716850296

There will be a subsection on this in the "images" section of the user manual once 5.0 is released.
